Articles/Blog/2023-03-08-productivity-practices-and-tools.md

```diff
@@ -0,0 +1,155 @@
+# Busy != Productive
+
+#### Contributed by [Robert Underwood](https://robertu94.github.io) and [Mark C. Miller](https://github.com/markcmiller86)
+
+We've all heard the slogans...*"Do more with less"*, *"Work smarter not harder"*, *"Failing to plan means planning to fail"*, *"Slower is faster"*.
+
+Some critical skills *effective* software professionals need to master include managing a wide variety of tasks, accounting for how they spend their time, and continuously improving their own practices.
+For many, journaling, task management, and time tracking tools will help with these practices.
+
+However, as David Sparks and Mike Schmitz hosts of the [Focused podcast](https://www.relay.fm/focused) put it, "life is about more than cranking widgets".
+The goal of these tools is not to enable getting more done (they probably will), but to enable people to be better stewards of their time and resources.
+While it may be satisfying to write another unit test, release a new feature, or resolve a stubborn bug, there is always more we *could* be doing.
+Mastering the practices and tools described here will help make more time available for what we believe we *should* be doing, most.
+
+### Productivity Journaling
+
+Productivity journaling is different from general journaling (e.g. keeping a diary).
+Productivity journaling captures tasks, observations, ideas, and lessons to be periodically reviewed, distilled and migrated.
+
+* **Tasks**: identify something that needs to be done (e.g. a bug to fix).
+* **Observations**: describe something that happened worth remembering almost like in a lab journal.
+* **Ideas**: capture unspecific but maybe important thoughts to be processed later (e.g. a better approach to a sorting problem).
+* **Lessons**: record key takeaways worth sharing with others (even your future self).
+
+Effective productivity journaling also demands that one periodically slow down, pause, reflect and review their record of activity.
```

Comment:
```suggestion
Effective  journaling also demands that one periodically slow down, pause, and reflect to review their record of activity.
```

small grammar thing: a of verbs has to have the same actor and object if used in a conjunctive expression.  One does not slow down their record of activity